lib/chef/resource/windows_printer.rb

```diff
@@ -116,62 +116,62 @@ class WindowsPrinter < Chef::Resource
         introduced: "17.3",
         default_description: "The resource block name or the ipv4_address prepended with IP_."
 
+      def ps_bool_to_ruby(bool)
+        bool.to_s.downcase == "true" ? true : false
```

Comment:
you don't need the ternary here at all.

Reply:
Then a "false" would end up being a nil in the resource which would blow up the type validation

Reply:
`bool.to_s.downcase == "true"` is only going to be true or false